src/components/pages/MentorMenteeMatching/MentorOptions.js

```diff
@@ -26,19 +89,6 @@ export const select = (edit, setMentors, mentors, data, name, nameVal) => {
   );
 };
 
-export const options = data => {
-  const { Option } = Select;
-
-  return (
-    <>
-      <Option value="Assign Mentor">Assign Mentor</Option>
-      <Option value={data.first}>{data.first}</Option>
-      <Option value={data.second}>{data.second}</Option>
-      <Option value={data.third}>{data.third}</Option>
-    </>
-  );
-};
-
```

Comment:
Good work on eliminating the hard coded drop down in favor of the programmatic approach.